src/main/java/org/opentripplanner/middleware/tripMonitor/jobs/CheckMonitoredTrip.java

```diff
@@ -46,11 +45,9 @@
 public class CheckMonitoredTrip implements Runnable {
     private static final Logger LOG = LoggerFactory.getLogger(CheckMonitoredTrip.class);
 
-    private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern(
-        DateTimeUtils.DEFAULT_DATE_FORMAT_PATTERN
-    );
-
     public final MonitoredTrip trip;
+    public int departureDelay;
+    public int arrivalDelay;
```

Comment:
Where did these come from? They aren't used anywhere.

Reply:
Probably copy/paste. I removed it in https://github.com/ibi-group/otp-middleware/pull/102/commits/98f91c7d7043e0039a39a1855dc3e40befbf178e.